addons/mail/static/src/core/common/core.scss

```diff
@@ -46,12 +46,19 @@
     z-index: 1;
 }
 
-a.o_mail_redirect, a.o_channel_redirect {
-    @include button-variant(rgba($primary, .2), rgba($primary, .2), darken($link-color, 5%), rgba($primary, .3), rgba($primary, .3), darken($link-color, 10%));
+a.o_mail_redirect, a.o_channel_redirect, a.o-discuss-mention {
```

Comment:
I'm surprised to see we had tags in there, as we usually don't specify the tag (a in this case) in the style, as it makes it harder to maintain (tag is not really important to us, button would work the same for example) as well as adding css specificity.

But maybe for this specific case there was a reason that I don't remember.

Reply:
I don't think `a` is useless neither